nix/patches/iree/fix_alignment.patch

```diff
@@ -0,0 +1,30 @@
+diff --git a/runtime/src/iree/hal/drivers/local_sync/sync_device.c b/runtime/src/iree/hal/drivers/local_sync/sync_device.c
+index c65469b334..f0f7659055 100644
+--- a/runtime/src/iree/hal/drivers/local_sync/sync_device.c
++++ b/runtime/src/iree/hal/drivers/local_sync/sync_device.c
+@@ -6,6 +6,7 @@
+ 
+ #include "iree/hal/drivers/local_sync/sync_device.h"
+ 
++#include <stdalign.h>
+ #include <stddef.h>
+ #include <stdint.h>
+ #include <string.h>
+@@ -87,7 +88,7 @@ iree_status_t iree_hal_sync_device_create(
+       sizeof(*device) + loader_count * sizeof(*device->loaders);
+   iree_host_size_t total_size = struct_size + identifier.size;
+   iree_status_t status =
+-      iree_allocator_malloc(host_allocator, total_size, (void**)&device);
++      iree_allocator_malloc_aligned(host_allocator, total_size, alignof(iree_hal_sync_device_t), 0, (void**)&device);
```

Comment:
It's OK to keep these patches temporally. In the long time, you may raise an issue & send a PR to upstream.

Reply:
https://github.com/iree-org/iree/pull/20773